### DISCUSSION TITLE
🚀 Feature: Add Identicons to Avatars

### DISCUSSION THREAD

**Discussion by ItzNotABug** [appwrite/appwrite#11514] [Ideas]
### 🔖 Feature description

Avatars API currently has User Initials support but would be great to use randomised Identicons based on given input.

Additional feature could be to allow customising the identicon's color and background color.

As initially discussed on Discord:
https://discord.com/channels/564160730845151244/870534256324124692/1093262955010850998

More Info on Identicons:
https://en.m.wikipedia.org/wiki/Identicon

### 🎤 Pitch

```swift
let identicon = try await avatars.getIdenticon(identiconColor, backgroundColor)
```

where `identiconColor` and `backgroundColor` could be optional.
If nothing is passed, the api would generate random color for background and a white identicon.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by joeyouss** [comment]
Hi, Thank you for opening this.
I will leave this issue open and allow the community to leave their upvotes to know if this feature is indeed needed by them.

For now, let's leave this issue open

**Comment by ItzNotABug** [comment]
Circling back on this!
I've found [DiceBear](https://www.dicebear.com/styles) which offers diverse `Identicon` & `Avatar` styles. It could be a great reference or inspiration for the Avatars API. Worth a look!
